src/tasks/appveyor.js

```diff
@@ -41,7 +41,7 @@ module.exports = (config) => {
         'npm install',
       ],
       before_test: [
-        'cmd: npm install webpack@^%webpack_version%',
+        'cmd: npm install webpack@%webpack_version%',
```

Comment:
Oops, I missed off the other half of this PR, to change the webpack version to `latest` from a fixed version 😄